airbyte-integrations/connectors/destination-bigquery-denormalized/src/test/java/io/airbyte/integrations/destination/bigquery/JsonSchemaFormatTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class JsonSchemaFormatTest {
+
+  @Test
+  void fromJsonSchemaFormat_matchByFormatAndType() {
+    JsonSchemaFormat result = JsonSchemaFormat.fromJsonSchemaFormat("date-time", "timestamp_with_timezone");
+    assertEquals(JsonSchemaFormat.DATETIME_WITH_TZ, result);
+  }
+
+  @Test
+  void fromJsonSchemaFormat_matchByFormat() {
+    JsonSchemaFormat result = JsonSchemaFormat.fromJsonSchemaFormat("date", null);
+    assertEquals(JsonSchemaFormat.DATE, result);
+  }
+
+  @Test
+  void fromJsonSchemaFormat_notExistingFormat() {
+    JsonSchemaFormat result = JsonSchemaFormat.fromJsonSchemaFormat("not_existing_format", null);
+    assertNull(result);
```

Comment:
probably unrelated to your change, but I feel like this should default to string rather than null - that way we always have safe behavior, even for weird source schemas

Reply:
I think you are right but should I add this change in this PR?

Reply:
probably not - can you submit an issue for this?

Reply:
Done